src/operators/models.jl

```diff
@@ -11,11 +11,12 @@ function square_lattice_heisenberg(
     ::Type{T}=ComplexF64; Jx=-1, Jy=1, Jz=-1
 ) where {T<:Number}
     physical_space = ComplexSpace(2)
+    lattice = fill(physical_space, 1, 1)
     σx = TensorMap(T[0 1; 1 0], physical_space, physical_space)
     σy = TensorMap(T[0 im; -im 0], physical_space, physical_space)
     σz = TensorMap(T[1 0; 0 -1], physical_space, physical_space)
     H = (Jx * σx ⊗ σx) + (Jy * σy ⊗ σy) + (Jz * σz ⊗ σz)
-    return NLocalOperator{NearestNeighbor}(H / 4)
+    return nearest_neighbour_hamiltonian(lattice, H)
```

Comment:
```suggestion
    return nearest_neighbour_hamiltonian(lattice, H / 4)
```
Accidentally dropped the division, so the Heisenberg test is failing.

Reply:
I'm super confused, I noticed this this morning, pushed a fix, which shows up [here](https://github.com/QuantumKitHub/PEPSKit.jl/tree/ld-expval), but not in the PR

Reply:
ok, I forced the matter by changing the base branch a bit, seems ok now